cypress_shared/helpers/index.ts

```diff
@@ -47,7 +47,16 @@ const riskToSelfSummariesFromApplication = (
 }
 
 const tableRowsToArrays = (tableRows: Array<TableRow>): Array<Array<string>> => {
-  return tableRows.map(row => Object.keys(row).map(i => (row[i].html ? Cypress.$(row[i].html).text() : row[i].text)))
+  return tableRows.map(row => Object.keys(row).map(i => uiObjectValue(row[i])))
+}
+
+const uiObjectValue = (tableCell: TableCell) => {
```

Comment:
Oh, this is _very_ useful! I think I did a similar thing in my PR, but didn't extract it to a helper